faker/providers/person/ja_JP/__init__.py

```diff
@@ -31,16 +32,59 @@ class Provider(PersonProvider):
 
     first_names = first_names_male + first_names_female
 
-    # link: https://dic.nicovideo.jp/a/日本の苗字(名字)の一覧
-    last_names = (
-        '青田', '青山', '石田', '井高', '伊藤', '井上', '宇野', '江古田',
-        '大垣', '加藤', '加納', '喜嶋', '木村', '桐山', '工藤', '小泉',
-        '小林', '近藤', '斉藤', '坂本', '佐々木', '佐藤', '笹田', '鈴木',
-        '杉山', '高橋', '田中', '田辺', '津田', '中島', '中村', '渚',
-        '中津川', '西之園', '野村', '原田', '浜田', '廣川', '藤本', '松本',
-        '三宅', '宮沢', '村山', '山岸', '山口', '山田', '山本', '吉田',
-        '吉本', '若松', '渡辺',
-    )
+    # based on: https://www2.nipponsoft.co.jp/bldoko/index.asp
+    last_names = OrderedDict((
+        "佐藤", 366803,
```

Comment:
You need to wrap each name/probability (ie: each row) in parenthesis. For an example, see https://github.com/joke2k/faker/blob/master/faker/providers/person/en_NZ/__init__.py#L24